dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentitydatavalue/TrackedEntityDataValueChangeLogService.java

```diff
@@ -39,6 +39,10 @@ public interface TrackedEntityDataValueChangeLogService {
   void addTrackedEntityDataValueChangeLog(
       TrackedEntityDataValueChangeLog trackedEntityDataValueChangeLog);
 
+  /**
+   * @deprecated use ChangeLogService.getEventChangeLog(UID) instead
```

Comment:
Can we use a `{@link to class}` in here? This way its easy to navigate and will easily be updated on refactorings.

Reply:
That was my idea, but I couldn't find a way to link to a class in a different module, is that even possible?

Reply:
ah sorry, should be but not here as the dhis-api does not depend on service-tracker but the other way round